hudi-aws/src/main/java/org/apache/hudi/aws/transaction/lock/S3StorageLockClient.java

```diff
@@ -226,25 +229,43 @@ private static Functions.Function2<String, Properties, S3Client> createDefaultS3
         requiredFallbackRegion = true;
       }
 
-      S3Client s3Client = S3Client.builder().credentialsProvider(
-              HoodieAWSCredentialsProviderFactory.getAwsCredentialsProvider(props)).region(region).build();
+      // Set all request timeouts to be 1/5 of the default validity.
+      // Each call to acquire a lock requires 2 requests.
+      // Each renewal requires 1 request.
+      long defaultValiditySeconds = VALIDITY_TIMEOUT_SECONDS.defaultValue();
+      if (props.contains(VALIDITY_TIMEOUT_SECONDS.key())) {
+        defaultValiditySeconds = (long) props.get(VALIDITY_TIMEOUT_SECONDS.key());
+      }
+      long configuredS3TimeoutSeconds = defaultValiditySeconds / 5;
```

Comment:
minor. for longer variables, you can use "secs" as unit. 
also, we can just name the var as "s3CallTimeoutSecs"